bothub/project/handle.py

```diff
@@ -1,6 +1,8 @@
 from amqp.channel import Channel
 
 from .consumers.project_consumer import ProjectConsumer
+from .consumers.template_type_consumer import TemplateTypeConsumer
 
```

Comment:
According to PEP-8, it is recommended to use 2 blank lines after imports, I suggest you follow these recommendations to comply with the standards.

reference: https://peps.python.org/pep-0008/#blank-lines